src/stdplus/eint.hpp

```diff
@@ -82,6 +82,75 @@ T eint_plus(T a, T b, typename std::enable_if_t<std::is_integral<T>::value>* = 0
 	return a+b;
 }
 
+template <typename Ta, typename Tb>
+bool eint_less(Ta a, Tb b, typename std::enable_if_t<std::is_integral<Ta>::value && std::is_integral<Tb>::value>* = 0
+							, typename std::enable_if_t<std::is_unsigned<Ta>::value>* = 0
+							, typename std::enable_if_t<std::is_signed<Tb>::value>* = 0) {
+	if (b<0) return false;
+	else {
+		if (sizeof(Ta) >= sizeof(Tb)){
```

Comment:
maybe better do compare
std::numeric_limits<S>::digits of them
or something like that